launch/base_hardware.launch

```diff
@@ -16,7 +16,7 @@
             <arg name="bond_id_namespace" value="$(arg bond_id_namespace)" />
```

Comment:
This file should really take a `platform` argument instead of a `sim` argument

Reply:
Fixed